embulk-core/src/main/java/org/embulk/spi/util/ReporterPlugins.java

```diff
@@ -0,0 +1,58 @@
+package org.embulk.spi.util;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import org.embulk.config.ConfigSource;
+import org.embulk.config.TaskSource;
+import org.embulk.spi.Reporter;
+import org.embulk.spi.ReporterPlugin;
+
+import java.util.Map;
+
+public class ReporterPlugins
```

Comment:
Thank you for suggesting. Agree. Removed such utilities and moved such methods to `BulkLoader` on 7b3db91e9d67492e792de37bfc48fb20aa8e31be and b5a71239d4977780327e039179d129102cfc95ce